packages/synapse-react-client/src/synapse-queries/entity/useExportToTerra.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  DownloadPFBRequest,
+  QueryBundleRequest,
+} from '@sage-bionetworks/synapse-types'
+import { displayToast } from '../../components/ToastMessage/ToastMessage'
+import { parseEntityIdFromSqlStatement } from '../../utils/functions/SqlFunctions'
+import { useDownloadTableQueryResultAsPFB } from '../table/useDownloadTable'
+
+export function useExportToTerra(queryBundleRequest: QueryBundleRequest) {
+  const { mutateAsync: createPfb } = useDownloadTableQueryResultAsPFB()
+  // TODO?: SQL may need to be modified (see CAVATICA example) to include fields such as ga4gh_drs_uri
```

Comment:
Does John or Bruce know if they expect this DRS column in the PFB manifest?  If it is required, it may be best to refactor the CAVATICA sql for reuse while you're creating this new hook

Reply:
I have no idea, but I agree that it's a good opportunity to refactor if there is overlap. I want to track the integration implementation in new ticket(s)